src/app/admin/auditLogTable/sortFunctions.ts

```diff
@@ -23,7 +23,7 @@ export const auditLogTableSortableColumns: SortOptions<AuditLogRow>[] = [
         key: "actorName",
         sortMethodConfig: {
             method: (query, sortDirection) =>
-                query.order("actor_profile_name", { ascending: sortDirection === "asc" }),
+                query.order("actor_name", { ascending: sortDirection === "asc" }),
```

Comment:
Why do you think this bug happened in the first place? Can we be strict about what the column can be so typescript can pick this up?

I did a little experiment and this gives me an error successfully, so I think you can incorporate this into how filters are configured? (can make a separate ticket to do this across the project if you prefer)
```ts
function aaaa(query: PostgrestFilterBuilder<Database["public"], AuditLogPlusRow, any>): void {
    query.order<keyof AuditLogPlusRow>("a");
}
```

Reply:
sure this would be good to do, i'll make a separate ticket so we can get this merged but happy to assign it to myself :)

Reply:
https://softwiretech.atlassian.net/jira/software/c/projects/VFB/boards/862?selectedIssue=VFB-211

Reply:
thanks!